package.json

```diff
@@ -105,7 +105,7 @@
     "style-loader": "^1.3.0",
     "stylelint": "^13.13.1",
     "svg-sprite-loader": "^6.0.11",
-    "svgo-loader": "^2.2.2",
+    "svgo-loader": "~3.0.1",
```

Comment:
As discussed, let's be consistent with using `^` or `~` with packages that exist in combination. Otherwise we might end up in an even stranger place later.